jerry-core/jmem/jmem-allocator.c

```diff
@@ -221,6 +225,10 @@ jmem_decompress_pointer (uintptr_t compressed_pointer) /**< pointer to decompres
 #if defined (ECMA_VALUE_CAN_STORE_UINTPTR_VALUE_DIRECTLY) && defined (JERRY_CPOINTER_32_BIT)
   JERRY_ASSERT (uint_ptr % JMEM_ALIGNMENT == 0);
 #else /* !ECMA_VALUE_CAN_STORE_UINTPTR_VALUE_DIRECTLY || !JERRY_CPOINTER_32_BIT */
+#ifdef JERRY_ENABLE_EXTERNAL_CONTEXT
+  JERRY_DEFINE_CURRENT_CONTEXT ();
+  JERRY_UNUSED (jerry_current_external_context);
+#endif
```

Comment:
Ditto. (If external context is not enabled, then `JERRY_DEFINE_CURRENT_CONTEXT` is empty, so the `JERRY_UNUSED` would not be needed. But if external context is enabled, then the next line is using `JERRY_HEAP_CONTEXT`, which will use `JERRY_CONTEXT`, whic will use `jerry_current_external_context`. So, again, the unused macro should not be necessary. But then the whole `JERRY_ENABLE_EXTERNAL_CONTEXT` guard should become superfluous.)

Note: Should this guarding still be necessary for some reason, then style guide requires proper `#endif /* JERRY_ENABLE_EXTERNAL_CONTEXT */` commenting.